designs/replicated-pv/mayastor/encryption-at-rest.md

```diff
@@ -0,0 +1,109 @@
+---
+oep-number: OEP 1809
+title: OpenEBS Enhancement Proposal for Mayastor At-Rest Encryption of data
```

Comment:
Let's include Mayastor on the PR title as well to make it clear this is targeting mayastor only